Makefile

```diff
@@ -70,15 +74,14 @@ endif
 	@./tests/validate/whitespace.sh
 	@./tests/validate/govet.sh
 	@./tests/validate/git-validation.sh
-	@./tests/validate/gometalinter.sh . cmd/buildah
+	lint
```

Comment:
Since this is a make target I guess it won't work here. We could add it as dependency to `validate: lint`